yourgov/components/Staff/StaffMembersTable.tsx

```diff
@@ -24,10 +28,44 @@ type StaffMembersTableProps = {
 
 const headingId = 'staff-members-heading';
 
+const successTypeToMessageText = {
+	invite: {
+		description: 'will receive an email asking them to create a staff account.',
```

Comment:
This sentence feels weird and kinda unnecessary. The title says an email has been sent, and then this says an email will be sent

Reply:
Yep, it's also the only one with a description.